.github/workflows/build-lint-test.yml

```diff
@@ -74,40 +74,14 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        node-version: [18.x, 20.x, 22.x]
```

Comment:
wow, GH made this look so weird.

The actual change is the removal of the `compatibility-test:` job